deployment/scripts/chainspec/testnet/state_config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "coins": [],
+  "messages": [],
+  "contracts": [
+    {
+      "contract_id": "7777777777777777777777777777777777777777777777777777777777777777",
+      "code": "9000000994318e6e453f30e85bf6088f7161d44e57b86a6af0c955d22b353f91b2465f5e6140000a504d00205d4d30001a4860004945048076440001240400005050c0043d51345024040000",
```

Comment:
The address of the fee recipient is the same for now and will be changed in a separate PR.